README.md

```diff
@@ -34,7 +42,110 @@ Or install it yourself as:
 
     $ gem install transproc
 
-## Usage
+## Basics
+
+The simple transformations are defined as easy as
```

Comment:
Would drop `The` from the start of this sentence and just go with `Simple transformations are defined as easy as:`